### DISCUSSION TITLE
SAML integration removal without access

### DISCUSSION THREAD

**Discussion by danielapto** [community/community#61272] [New to GitHub]
How do we get help removing a SAML integration?

Our SAML IDP is down and we do not have our recovery codes.

**Comment by queenofcorgis** [comment]
Hi there @danielapto 👋🏼 

This [doc](https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-saml-single-sign-on-for-your-organization/disabling-saml-single-sign-on-for-your-organization) takes you step by step through disabling, however if you don't have access you will need to open a ticket on our [Support page](https://support.github.com/contact) for account-specific assistance.
